scripts/integration/docker-compose.http-scrape.yml

```diff
@@ -0,0 +1,75 @@
+version: "3"
+
+services:
+  # https://github.com/sigoden/dufs
+  # serves static files at an HTTP endpoint
+  dufs:
+    image: docker.io/sigoden/dufs:latest
+    networks:
+      - backend
+    command:
+      - "/data"
+    volumes:
+      - ${PWD}/tests/data/http-scrape/:/data
+  # To validate Basic HTTP authentication option
+  dufs-auth:
+    image: docker.io/sigoden/dufs:latest
+    networks:
+      - backend
+    command:
+      - "-a"
+      - "/@user:pass"
+      - "--auth-method"
+      - "basic"
+      - "/data"
+    volumes:
+      - ${PWD}/tests/data/http-scrape/:/data
+  # To validate TLS options
+  dufs-https:
+    image: docker.io/sigoden/dufs:latest
+    networks:
+      - backend
+    command:
+      - "--tls-cert"
+      - "/certs/ca.cert.pem"
+      - "--tls-key"
+      - "/certs/ca.key.pem"
+      - "/data"
+    volumes:
+      - ${PWD}/tests/data/http-scrape/:/data
+      - ${PWD}/tests/data/ca/certs/ca.cert.pem:/certs/ca.cert.pem
+      - ${PWD}/tests/data/ca/private/ca.key.pem:/certs/ca.key.pem
+  runner:
+    build:
+      context: ${PWD}
+      dockerfile: scripts/integration/Dockerfile
+      args:
+        - RUST_VERSION=${RUST_VERSION}
+    working_dir: /code
+    networks:
+      - backend
+    command:
+      - "cargo"
+      - "nextest"
+      - "run"
+      - "--no-fail-fast"
+      - "--no-default-features"
+      - "--features"
+      - "http-scrape-integration-tests"
+      - "--lib"
+      - "sources::http_scrape::scrape::"
+    depends_on:
+      - dufs
+      - dufs-auth
+      - dufs-https
+    volumes:
+      - ${PWD}:/code
```

Comment:
I saw that `chronicle` does not, so seemed like it wasn't necessary? 🤷 

```
$ grep -riL --include="docker-compose.*.yml" target
docker-compose.http-scrape.yml
docker-compose.chronicle.yml
docker-compose.axiom.yml
```

Reply:
Yeah, not sure if we should/need to. I suspect it's not particularly helpful, especially for mac users since the code under `target/` wouldn't be usable in the linux container.

Reply:
The integration tests that I have so far pass as is without it so unless otherwise suggested I'll leave it out.

Reply:
Yeah, I dropped all mounting of `target/` to alleviate permissions issues.

https://github.com/vectordotdev/vector/pull/12977

Reply:
Oh so it sounds like it should actually be _added_ to these then:

```
docker-compose.http-scrape.yml
docker-compose.chronicle.yml
docker-compose.axiom.yml
```